HinojosaFernando/src/Library.java

```diff
@@ -0,0 +1,121 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+
+public class Library {
+    private Person owner;
+    private Map<String, Book> bookMap;
+    private Map<String, Movie> movieMap;
+    private Map<Genres, List<String>> byGenre;
+    private Map<String, Person> creatorCollection;
+    private String row;
+
+    public Library() {
+        bookMap = new HashMap<>();
+        movieMap = new HashMap<>();
+        byGenre = new HashMap<>();
+        loadBooks();
+        loadMovies();
+    }
+
+    public void addBook(Book book) {
+        bookMap.put(book.getTitle(), book);
+        if (!byGenre.containsKey(book.getGenres())) {
+            byGenre.put(book.getGenres(), new ArrayList<>());
+            //byGenre.get(book.getGenres()).add(book.getTitle());
+        }
+        byGenre.get(book.getGenres()).add(book.getTitle());
+    }
+
+    public List<String> getAllStoryByGenere(Genres gen) {
+
+        if (byGenre.containsKey(gen)) {
+            return byGenre.get(gen);
+        } else {
+            System.out.println("Genres is not present");
+            return null;
+        }
+
+    }
+
+    public void addMovie(Movie movie) {
+        movieMap.put(movie.getTitle(), movie);
+        if (!byGenre.containsKey(movie.getGenres())) {
+            byGenre.put(movie.getGenres(), new ArrayList<>());
+        }
+        byGenre.get(movie.getGenres()).add(movie.getTitle());
+    }
+
+    public void loadMovies() {
+        BufferedReader readerMovie = null;
+        Movie loadMovies;
+        try {
+            readerMovie = new BufferedReader(new FileReader("/home/fernando_hinojosa/Documents/Story/movie.csv"));
```

Comment:
So here you only need to call this constructor with the String that will passed to this method.